src/paulscode/android/mupen64plusae/MenuListView.java

```diff
@@ -222,10 +269,23 @@ public View getGroupView( int groupPosition, boolean isExpanded, View convertVie
                 TextView text1 = (TextView) view.findViewById( R.id.text1 );
                 TextView text2 = (TextView) view.findViewById( R.id.text2 );
                 ImageView icon = (ImageView) view.findViewById( R.id.icon );
+                ImageView indicator = (ImageView) view.findViewById( R.id.indicator );
                 
                 text1.setText( item.getTitle() );
                 text2.setVisibility( View.GONE );
                 icon.setImageDrawable( item.getIcon() );
+                
+                if ( item.isChecked() )
+                    view.setBackgroundColor( 0x44FFFFFF );
```

Comment:
Transparent background color allows the keyboard selection color to show through.